Kickstarter-iOS/Views/Controllers/MessageBannerViewController.swift

```diff
@@ -57,9 +57,8 @@ final class MessageBannerViewController: UIViewController {
     self.viewModel.outputs.iconTintColor
       .observeForUI()
       .observeValues { [weak self] color in
-        guard let `self` = self else { return }
-        _ = self.iconImageView
-          |> UIImageView.lens.tintColor .~ color
+        _ = self?.iconImageView
+          ?|> UIImageView.lens.tintColor .~ color
```

Comment:
Here I would use keyPaths, since the plan is to deprecate lenses in the future.

Reply:
Whoops I missed this. Will do!